tauri/src/endpoints/asset.rs

```diff
@@ -19,7 +20,7 @@ pub fn load(
       });
       let mut read_asset;
       loop {
-        read_asset = crate::assets::ASSETS.get(&format!(
+        read_asset = assets.get(&format!(
           "{}/{}",
           option_env!("TAURI_DIST_DIR")
             .expect("tauri apps should be built with the TAURI_DIST_DIR environment variable"),
```

Comment:
*notes for myself:*
This needs to be adapted.  See `runner.rs` + `server.rs` for how they handle it now for embedded server assets.  It might be as simple as changing it to `format!("/{}", path.to_string_lossy())` since the keys are already normalized to have a `/` to start them and to use unix/url paths.  If the path passed to this can possibly have windows endings, we may need to move our `to_key` function from `tauri-marcos` to `tauri-api::assets` so that it can be used from proc-macros and this code.